packages/storybook/src/generators/init/init.ts

```diff
@@ -43,6 +49,10 @@ function checkDependenciesInstalled(host: Tree, schema: Schema) {
     devDependencies['@storybook/manager-webpack5'] = storybookVersion;
   }
 
+  if (packageManager === 'npm') {
```

Comment:
I think we just install it regardless because we can't rely on hoisting the dependency if the workspace really needs it.

Let's add a TODO to remove this once Storybook v7 is out and we no longer need webpack.

Reply:
Ok, will remove the check!